File: 프로그래머스0(61~90번)/80.5명씩.js
// 레벨 0
// https://school.programmers.co.kr/learn/courses/30/lessons/181886

/*
앞에서 부터 5명씩 묶은 그룹의 가장 앞에 서있는 사람들의 이름을 
담은 리스트를 return하도록 solution 함수를 완성해주세요. 
*/

function solution(names) {
    var answer = [];
    
    for (let i=0; i<names.length; i++) {
        if (i % 5 === 0) {
            //answer = answer.push[i];
            answer.push(names[i]);
        }
    }
    
    return answer;
}

File: 프로그래머스 코테(java)/프로그래머스0(61~90번)/80.5명씩.java
// 레벨 0
// https://school.programmers.co.kr/learn/courses/30/lessons/181886

import java.util.ArrayList;

class Solution {

    public String[] solution(String[] names) {

        ArrayList<String> list = new ArrayList();
        int idx = 1;
        for (int i = 0; i < names.length; i++) {
            if (i % 5 == 0) {
                list.add(names[i]);
            }
        }
        String[] answer = new String[list.size()];

        for (int i = 0; i < list.size(); i++) {
            answer[i] = list.get(i);
        }

        return answer;
    }
}